server/server.js

```diff
@@ -41,4 +41,15 @@ app.post('/api/hostels/:hostel_id/images', (req, res) => {
   });
 });
 
+app.delete('/api/hostels/:hostel_id/images', (req, res) => {
+  console.log(req.body);
+  models.image.deleteEntry(req.body.id, (error, deletedImage) => {
```

Comment:
Can the id be pulled from the URL params or do they need to be included in the body? The body makes sense for creating but deleting could just use the id. 

What if there was an endpoint for deleting a single image AND one for deleting all images in a hostile?